Sources/CoffeeAddictsLibrary/Utils/ParsingStatus.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mihai Garda Popescu on 23.07.2021.
+//
+
+import Foundation
+
+enum ParsingStatus: String {
```

Comment:
There is a read flag the pops up as soon as I see folders like `utils` or `helpers` or `managers` ...

in your case I think `Constants` is fine (check out my general comment) but this enum might be better in another place, somewhere closer to the parser.

Reply:
> There is a read flag the pops up as soon as I see folders like `utils` or `helpers` or `managers` ...
> 
> in your case I think `Constants` is fine (check out my general comment) but this enum might be better in another place, somewhere closer to the parser.

Besides my general comment, I agree that this could reside in the CSVParser itself or the the file could go in a separate folder together with the parser :)

I used to work on a project with two dozen enums and it was pretty difficult to debug and navigate through them because they were in the same files with other logic. 

So it’s just a reminiscence of that, I don’t hold to it or against it.